_ingest-pipelines/processors/ml-inference.md

```diff
@@ -0,0 +1,158 @@
+| `description` | String | Optional | A brief description of the processor.                                                                                                                                                                                                                                                                                                                                                                             |
+| `tag` | String | Optional | An identifier tag for the processor. Useful for debugging to distinguish between processors of the same type.                                                                                                                                                                                                                                                                                                     |
+| `ignore_missing` | Boolean | Optional | If true and any of the input fields defined in input_map or output_map are missing, those missing fields are quietly ignored. Otherwise, a missing field causes a failure. Default is false.                                                                                                                                                                                                                      |
+| `ignore_failure` | Boolean | Optional | If true and any exceptions occur, they are quietly ignored, and the ingestion continues. Otherwise, an exception can occur and abort the ingestion. Default is false.                                                                                                                                                                                                                                             |
+
+## Using the processor
+
+Follow these steps to use the processor in a pipeline. You must provide a model ID when creating the processor. For more information, see [Using custom models within OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/using-ml-models/).
+
+**Step 1: Create a pipeline.**
+
+The following example request creates an ingest pipeline where the model requires an `input` field and produces a `data` field in the prediction task and the document has a `passage_text` field, which  will be converted into text embeddings and the embeddings will be stored in `passage_embedding` field. 
+
+The mappings for input_map and output_map also support standard json path notation to complex data structures.
+
+```json
+
+PUT /_ingest/pipeline/ml_inference_pipeline
+{
+  "description": "Generate passage_embedding for ingested documents",
+  "processors": [
+    {
+      "ml_inference": {
+        "model_id": "nSQsK48BrpZG38I5Joby",
```

Comment:
Should we use `<your model id>` ? Or add some explanation before this example like "suppose you have a model with model id nSQsK48BrpZG38I5Joby"

Reply:
use <your model id>